deployments/liqo/values.yaml

```diff
@@ -62,6 +62,18 @@ gateway:
   config:
     # -- port used by the vpn tunnel.
     listeningPort: 5871
+  metrics: 
+    # -- enable liqo-gateway metrics server which expose metrics about cluster peers.
+    enabled: false
+    # -- port used by tunnel-operator's metrics server which expose VPN metrics.
+    port: 5872
+    serviceMonitor:
+      # -- create a prometheus service monitor for the tunnel-operator's metrics.
+      enabled: false
+      # -- setup service monitor requests interval.
+      interval: "1m"
+      # -- setup service monitor scrape timeout.
+      scrapeTimeout: "15s"
```

Comment:
Still applies the fact that a user does not know anything about what the tunnel-operator is.